app/models/pillar.rb

```diff
@@ -41,7 +41,9 @@ def all_pillars
         ldap_admin_group_name:   "ldap:admin_group_name",
         ldap_tls_method:         "ldap:tls_method",
         ldap_mail_attribute:     "ldap:mail_attribute",
-        cloud_framework:         "cloud:framework"
+        cloud_framework:         "cloud:framework",
+        suse_mirror_url:         "cloud:suse_mirror_url",
+        suse_mirror_certificate: "cloud:suse_mirror_certificate"
```

Comment:
No idea if these names are good or if it makes sense here.

Reply:
I'd rename them to something like:

```
ldap_mail_attribute:              "ldap:mail_attribute",
suse_registry_mirror_url:         "suse_registry_mirror:url",
suse_registry_mirror_certificate: "suse_registry_mirror:certificate",
cloud_framework:                  "cloud:framework"
```

And also its callers in `setup_controller.rb`. I'd leave `cloud_framework` the last one on the list at least for now.

Reply:
Will do.